### PR TITLE
added test for about page

### PR DESCRIPTION
**Pull request by dotslashbit**

### CODE REVIEW

File: src/__tests__/pages/About.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { screen, render } from "@testing-library/react";
+import About from "../../pages/About";
+import { MemoryRouter } from "react-router-dom";
+
+describe("About Page", () => {
+  test("renders without crashing", () => {
+    render(
+      <MemoryRouter>
```
Comment (anth-volk) [inline src/__tests__/pages/About.test.js:10]: This should actually be `BrowserRouter`, as we don't really use `MemoryRouter` in our code (that I know of)

File: src/__tests__/pages/About.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <About />
+      </MemoryRouter>,
+    );
+
+    const heading = screen.getByRole("heading", { name: /Our people/i });
```
Comment (anth-volk) [inline src/__tests__/pages/About.test.js:15]: I like the use of "getByRole" here. Being honest, I didn't even know that was a type of role.

File: src/__tests__/pages/About.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    expect(heading).toBeInTheDocument();
+    const max = screen.getByText(
+      "PolicyEngine's team leads a global movement of open-source contributors.",
+    );
+    expect(max).toBeInTheDocument();
+
+    screen.debug(max);
```
Comment (anth-volk) [inline src/__tests__/pages/About.test.js:23]: Just to flag: best to remove this line before pushing the test

File: src/__tests__/pages/About.test.js (hunk 1 above)
Comment (dotslashbit) [reply src/__tests__/pages/About.test.js:10]: got it

File: src/__tests__/pages/About.test.js (hunk 2 above)
Comment (dotslashbit) [reply src/__tests__/pages/About.test.js:15]: Thanks, When I was learning about the query types, it was written that getbyrole is always preferable.

File: src/__tests__/pages/About.test.js (hunk 3 above)
Comment (dotslashbit) [reply src/__tests__/pages/About.test.js:23]: got it

**Review by anth-volk — APPROVED** [review]
Thanks @dotslashbit, this is perfect! If you're ready to go with this, if you could convert out of draft and run the linter, we can merge.

**Comment by dotslashbit** [comment]
@anth-volk I have added some more tests that we discussed on slack, please take a look, thanks for all the help and support.

**Review by anth-volk — APPROVED** [review]
Looking through this, this looks great, @dotslashbit! Thanks so much for your work, it's great to increase our testing.

**Pull request merged**